templates/common/user_tools.html

```diff
@@ -44,6 +44,10 @@ <h2 class="username">
       <ul id="user-actions" class="toolset clear">
 
         $if not relationship['is_self']:
+		  <li>
+		    $if relationship['followby']:
+			  <span class="icon icon-20 icon-followed"></span>Following You
+		  </li>
```

Comment:
This code doesn't follow the indentation in the rest of the file. I can't check right this moment, but I don't think the templates use tabs at all.